support-frontend/typescript-errors.json

```diff
@@ -788,6 +787,14 @@
             "path": "assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx",
             "message": "Parameter 'result' implicitly has an 'any' type."
         },
+        {
+            "path": "assets/pages/contributions-landing/contributionsLanding.tsx",
+            "message": "Parameter 'props' implicitly has an 'any' type."
+        },
+        {
+            "path": "assets/pages/contributions-landing/contributionsLanding.tsx",
+            "message": "Parameter 'props' implicitly has an 'any' type."
+        },
```

Comment:
I'm not quite sure why these have been introduced :/ maybe running yarn will fix them too?